<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TableToolbar</class>
 <widget class="QWidget" name="TableToolbar">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>97</width>
    <height>30</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QToolButton" name="btnAdd">
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset resource="../res/res.qrc">
       <normaloff>:/res/image/icon/addition.png</normaloff>:/res/image/icon/addition.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>24</width>
       <height>24</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="btnDel">
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset resource="../res/res.qrc">
       <normaloff>:/res/image/icon/delete.png</normaloff>:/res/image/icon/delete.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>24</width>
       <height>24</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QToolButton" name="btnClr">
     <property name="text">
      <string>...</string>
     </property>
     <property name="icon">
      <iconset resource="../res/res.qrc">
       <normaloff>:/res/image/icon/trash.png</normaloff>:/res/image/icon/trash.png</iconset>
     </property>
     <property name="iconSize">
      <size>
       <width>24</width>
       <height>24</height>
      </size>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="../res/res.qrc"/>
 </resources>
 <connections/>
</ui>
